### ISSUE TITLE
Add labels to CRDs in install .yaml files to prevent errors during install

### ISSUE THREAD

**Issue by samodell** [knative/docs#1036]
More background info in #968 , but tl;dr: Installs of multiple components sometimes fail due to race conditions happening in the way CRDs are applied. To fix this issue, we can label the CRDs in the install .yaml files so that we can control what is applied first. Goal is to have this in place for v0.5.

- [x] Serving 
- [x] Eventing
- [x] Build

**Comment by samodell** [comment]
A little more detail on the plan, from Evan in #docs on Slack:
"We're going to label all our CRDs, and then install the yaml with `kubectl -l resource_definition=true ....` the first time, then the second time without a label constraint."

**Comment by jamesward** [comment]
This is failing again on `0.7.0`

**Comment by samodell** [comment]
@jamesward  Could you paste the error messages you get here to help us trouble-shoot?

**Comment by jamesward** [comment]
```
error: unable to recognize "https://github.com/knative/eventing/releases/download/v0.7.0/release.yaml": no matches for kind "ClusterChannelProvisioner" in version "eventing.knative.dev/v1alpha1"
```

**Comment by samodell** [comment]
Thanks, @jamesward ! sometimes in the past these issues were resolved by re-running the same command a second time. Does that work in this case?

**Comment by jamesward** [comment]
yup! re-running it resolves the issue.

**Comment by carieshmarie** [comment]
Closing this issue as I believe it's resolved. Feel free to reopen if I'm incorrect.

**Comment by jamesward** [comment]
@carieshmarie Can you point me to the doc or product fix?

**Comment by carieshmarie** [comment]
I think following the instructions added in #1194 should prevent encountering the issue.

**Comment by jamesward** [comment]
Unfortunately those instructions no longer prevent the issue on 0.7.  Since it is a race condition, the issue is intermittent and the fix in #1194 no longer works consistently.
